content/post/value-reordering-lists.md

```diff
@@ -0,0 +1,238 @@
+[ensembling](https://dl.acm.org/citation.cfm?id=1835893) a variety of independent
+models can provide userful information by indicating why particular items were 
+ranked highly. If a recommended item was ranked highly by the Popularity submodel,
+you can put "Popular!" next to that item.
+These explanations can help users trust recommendations, 
+both by making clear that
+the ranking is built from factors that users believe to be causal, as 
+well as by "explaining away" recommendations that may not be useful to them. A churn model
+that indicates that Customer X is likely to not renew their contract, because there's 
+data that they're going out of business, is trust-worthy even when it's not actionable.
+
+Another thing to consider is
+under what circumstances should you provide a plain list, versus a list with numerical
+scores, versus a list with rank categories (high/med/low or red/yellow/green)?
+Suppose you're 
+trying to provide the FDNY a list of properties they should inspect. As the diagram
+above suggests, giving them the properties in ranked order is useful. Is it more
+helpful to provide numbers along with that ranking? The top location is 2.1% likely to
+catch on fire this year -- is that helpful? Or should you color-code by high/med/low risk,
+and if so, where do you draw the boundaries?
+
+At a [previous employer](http://eab.com), 
+we built a model of college students' likelihoods to fail to graduate,
+so that academic advisors can reach out and provide support and services. An interesting
+question that we tackled was how and when to present these risk scores. As of when I left 
+the company, 
+we were presenting several options to each college, and letting them choose based on their
+organization's
+culture and how they thought about risk and triage. 
+
+One option was to put an equal number
+of students in each of three groups. But for colleges with high graduation rates, that could
+put a large number of students who are still quite likely graduate in the "red" 
+category, which was unappealing. Another option was to fix the boundaries at 33% and 66%.
+But for colleges with low graduation rates, this meant that most of their students
+were in the "red" or "yellow" categories, which could be disheartening. I tended to recommend
+another option -- 
+boundaries of 50% and the overall graduation rate. So, for a college with a 75% 
+graduation rate, students with a less than 50% graduation rate would be red, those in the 
+50% to 75% range would be yellow, and 75% or higher (about half the population) would be green.
+Having grounded boundaries like this seemed to me to be a good way to help academic advisors make
+sense of this ranked and color-coded list. 
+That is, we were ordering a list using a well-calibrated model,
+then leveraging the underlying risk scores to provide intuitions about what list-position
+actually meant.
+
+# Final Thoughts
+
+(ugh, need some ideas for how to wrap this up...)
```

Comment:
The easiest way to wrap this up would just be a sign off sentence summarizing the points you covered and suggesting people think about ways folks can apply these ranking and sorting problems to their own businesses. Don't over-think it!